apps/web/pages/team/[slug]/book.tsx

```diff
@@ -2,9 +2,10 @@ import type { GetServerSidePropsContext } from "next";
 
 import type { LocationObject } from "@calcom/app-store/locations";
 import { privacyFilteredLocations } from "@calcom/app-store/locations";
+import { getBookingFieldsWithSystemFields } from "@calcom/features/bookings/lib/getBookingFields";
```

Comment:
In this page we can also load the dependencies that are only needed in getServerSideProps within the function, we must take care of the bundle size of the public pages.